context-and-compliance/03-environments.md

```diff
@@ -0,0 +1,33 @@
+# 3 - Environments
+
+## 3.1 Application has corresponding environments and hosting
+
+### 3.1.1 `testing` environment
+
+#### Context & justification
+
+The `testing` environment, as the name suggests, is an environment for testing. This is where all development/testing is done. Hence, it should only be accessible to engineers and for internal delivery teams. Self-explanatory.
+
+#### Compliance
+
+* [ ] `testing` for internal delivery team; this environment should only be accessible to engineers
+
+### 3.1.2 `staging` environment
+
+#### Context & justification
+
+The `staging` environment is an environment typically used for external (customer, users, etc) reviews. The staging environment may also require showcasing to customers depending on feedback from reviews, however it is also on a feature by feature basis. This environment is additionally used for high level end-to-end (E2E) testing, as part of **Acceptance Test Driven Development (ATDD)**, which tests from the input of the system to the output of the system.
```

Comment:
>The staging environment may also require showcasing to customers depending on feedback from reviews, however it is also on a feature by feature basis

No need to have the "it is also" part

>however this should be on a feature-by-feature basis

Reply:
Resolved in https://github.com/madetech/productionisation/pull/58/commits/109598ae69600f44cc6b160e1d22d55923d45045.